api/NetworkInformation.json

```
@@ -54,13 +54,16 @@
           "spec_url": "https://wicg.github.io/netinfo/#dom-networkinformation-downlink",
           "support": {
             "chrome": {
-              "version_added": "61"
+              "version_added": "61",
+              "notes": "The value is capped to 10Mbps as an anti-fingerprinting measure. This limit is not present in the specification."
```

style: I wasn't sure "capped" was particularly explicit (and potentially unclear to newcomers to English). While I was here, I tried giving that unit some room to breath and collapsed the second sentence into "non-standard".

What do you think?

```suggestion
              "notes": "The value is never greater than 10 Mbps, as a non-standard anti-fingerprinting measure."
```

---

Thanks - it is certainly more succinct. I will roll out to the other cases.